src/services/explore.ts

```diff
@@ -53,3 +57,40 @@ export function saveFilter(filter: any, name: string, token?: string) {
         token
     );
 }
+
+export function fetchElementsInfos(
+    ids: UUID[],
+    elementTypes?: string[],
+    equipmentTypes?: string[]
+) {
```

Comment:
```suggestion
): Promise<???> {
```
we don't want a `Promise<any>`.